frontend/resourceadm/pages/ResourcePage/ResourcePage.tsx

```diff
@@ -96,6 +97,12 @@ export const ResourcePage = (): React.JSX.Element => {
     }, 400);
   };
 
+  useEffect(() => {
```

Comment:
I can see that this is being used more than just this one place. Maybe it could be an idea to create a custom hook that can be reused in the files? 😄

Reply:
Good idea! I refactored this a bit further and changed the check for merge conflicts, and moved the code for this to resourceadm `PageLayout`